pkg/github/raw_file_service.go

```diff
@@ -0,0 +1,24 @@
+package github
+
+import (
+	"fmt"
+
+	"github.com/arquillian/ike-prow-plugins/pkg/scm"
+	"github.com/arquillian/ike-prow-plugins/pkg/utils"
+)
+
+// RawFileService encapsulates retrieval of files in the given GitHub repository change
+type RawFileService struct {
+	Change scm.RepositoryChange
+}
+
+// GetRawFile retrieves raw file content on the given path from the related GitHub repository change
+func (s *RawFileService) GetRawFile(path string) ([]byte, bool, error) {
+	return utils.GetFileFromURL(s.GetRawFileURL(path))
+}
+
+// GetRawFileURL creates a url to the given path related to the GitHub repository change
+func (s *RawFileService) GetRawFileURL(path string) string {
```

Comment:
Not sure why this should be public?

Reply:
because I'm using it for constructing URL when the custom-message-file is set as a relative path

Reply:
and for constructing URL to the config file

Reply:
both cases are for the comment purposes